### ISSUE TITLE
Battery not showing

### ISSUE THREAD

**Issue by JurajNyiri** [Barma-lej/landroid-card#7]
## **Before submitting a bug**

[x] I updated to the latest version available
[x] I cleared the cache of my browser

## **Describe the bug**

Battery is not showing in the card for me.

I am using landroid 2.0.3 of landroid_cloud because of https://github.com/MTrab/landroid_cloud/issues/199 .

Here are entity properties:

```
battery_level: 71
battery_icon: mdi:battery-charging-80
cloud_id: 286188
total_blade_time: 27936
current_blade_time: 27936
battery_voltage: 19.51
battery_temperature: 33.6
work_time: 29336
distance: 430349
yaw: 242.4
roll: -2.2
pitch: -0.1
mac: 98D863319B94
partymode_enabled: false
mowing_zone: 0
locked: true
online: true
zone: 0, 0, 0, 0
zone_probability: 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
schedule_enabled: true
schedules: 
primary:
  monday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  tuesday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  wednesday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  thursday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  friday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  saturday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  sunday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
secondary:
  monday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  tuesday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  wednesday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  thursday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  friday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  saturday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false
  sunday:
    start: '00:00'
    end: '00:00'
    duration: 0
    boundary: false

error: no error
total_charge_cycles: 180
current_charge_cycles: 180
last_update: 16:26:43 12/07/2022
rssi: -57
rain_delay: 0
rain_sensor_triggered: false
rain_delay_remaining: 0
firmware_version: 3.26
serial: 2019301993010020192B
time_extension: 0
error_id: 0
device_class: landroid_cloud__state
friendly_name: Mower
supported_features: 12500
```

## **Steps to reproduce**

1. Add card using following code:

```
image: default
compact_view: false
show_status: true
show_name: false
show_toolbar: true
type: custom:landroid-card
entity: vacuum.mower
stats:
  default:
    - attribute: total_blade_time
      subtitle: Total blade time
      value_template: '{{ as_timedelta((value | float(0) * 60) | string) }}'
    - attribute: current_blade_time
      subtitle: Current blade time
      value_template: '{{ as_timedelta((value | float(0) * 60) | string) }}'
    - attribute: work_time
      subtitle: Work time
      value_template: '{{ as_timedelta((value | float(0) * 60) | string) }}'
    - attribute: distance
      value_template: '{{ (value | float(0) / 1000) | round(3) }}'
      unit: km
      subtitle: Distance
  mowing:
    - attribute: yaw
      subtitle: Yaw
      unit: °
    - attribute: roll
      subtitle: Roll
      unit: °
    - attribute: pitch
      subtitle: Pitch
      unit: °
```

## **Expected behavior**

Battery showing just like in readme.

## **Screenshots**

![image](https://user-images.githubusercontent.com/5785999/178515170-f5b99c34-97f9-4510-be90-9714e5d5e6a2.png)

## **Versions:**

- Release: 0.1.4
- Browser: Chrome
- HomeAssistant version: 2022.07.03

## Additional notes

Thank you for this card, I like it a lot more than the last one, nice clean design!

**Comment by Barma-lej** [comment]
Will fixed in 0.1.5

**Comment by Skyber1967** [comment]
Seems to be fixed with landroid_cloud V2.2.1 so not a bug with landroid-card I think ;)

![image](https://user-images.githubusercontent.com/102285630/179348804-1c17a586-5ca9-4e07-a7e6-f02849296f3d.png)

**Comment by Barma-lej** [comment]
Fixed in 0.1.5
